### ISSUE TITLE
Override templates

### ISSUE THREAD

**Issue by rtrudel** [stefanvanherwijnen/quasar-app-extension-http-authentication#9]
Is there a way to override templates for Login, Register etc.?

As it's hard coded within the module, I would like to know if it's possible to configure something to use pages and components from src instead?

Thank you!

**Comment by stefanvanherwijnen** [comment]
Yes, just use a Vue file in your src/pages/auth folder: https://github.com/stefanvanherwijnen/quasar-app-extension-http-authentication/blob/master/test/app/src/pages/auth/Register.vue and define the route in your routes file. 

The AE will use that file instead (https://github.com/stefanvanherwijnen/quasar-app-extension-http-authentication/blob/2899010fc264546f3a853901d456dc5ebd141196/src/boot/index.js#L94)

**Comment by widefoot** [comment]
NavigationDuplicated {_name: "NavigationDuplicated", name: "NavigationDuplicated", message: "Navigating to current location ("/register") is not allowed", stack: "Error↵    at new NavigationDuplicated (webpack-int…node_modules/vue/dist/vue.runtime.esm.js:2178:14)"}

How I can fix it?

**Comment by stefanvanherwijnen** [comment]
Did you click on the register button while on the register page?
Vue by default returns an error if you navigate to the current location. Google is your friend for a fix.
If it's something else, let me know.

**Comment by stefanvanherwijnen** [comment]
Should be fixed or no longer applicable in v2.
